arch/arm64/src/imx9/imx9_clockconfig.c

```diff
@@ -58,6 +58,30 @@
  * Private Data
  ****************************************************************************/
 
+static uint32_t g_pll_freqs[] =
```

Comment:
add const

Reply:
This LUT should be writable, I will add support to change PLL frequency in the future and the selected frequency will be stored in this LUT.